vignettes/adrs.Rmd

```diff
@@ -235,6 +235,7 @@ the `{admiralonco}` parameter derivation functions in the below steps, via `sour
 However, if you prefer not to rely on `source_pd` arguments, then the user is free to filter out records after
 first `PD` at this stage in a similar way via a `ANLzzFL` flag, and then you could leave `source_pd` as null in
 all downstream parameter derivation function calls.
+So, for eg the user can create ANL02FL flag to subset the post baseline response data up to first reported progressive disease. The user can also add in anl02fl derivation to exclude any response data after start of any new anti-cancer therapy.This will be an alternative and transparent method to the use of source_pd argument approach to create ADRS parameters below.
```

Comment:
```suggestion
So, for example the user could create `ANL02FL` flag to subset the post-baseline response data up to and including first reported progressive disease. This would be an alternative and transparent method to the use of `source_pd` argument approach to create ADRS parameters below.
```

Reply:
i wouldn't mention NACT here again as we already give an example of that above.

Reply:
@rossfarrugia updated the text and added the example for derive_var_relative_flag(). But there are some checks failing.

Reply:
@amitjaingsk https://github.com/pharmaverse/admiral/pull/1502 needs to be merged first. will try and take a look at reviewing it for Stefan today. feel free to self-assign to that review too if you get chance

Reply:
@amitjaingsk `derive_var_relative_flag()` is merged now in admiral latest devel branch

Reply:
@rossfarrugia  thanks, I am still getting some eror due to some packages are out of sync. I have asked Matt to try to check from his system, so we will get you know.